modular_citadel/code/modules/mob/living/silicon/robot/dogborg_equipment.dm

```diff
@@ -348,7 +352,7 @@ SLEEPER CODE IS IN game/objects/items/devices/dogborg_sleeper.dm !
 		pixel_y = 10
 		update_icons()
 		throw_at(A, MAX_K9_LEAP_DIST, 1, spin=0, diagonals_first = 1)
-		cell.use(500) //Doubled the energy consumption
+		cell.use(750) //Same as stunbaton

```

Comment:
Batons use 1000 charge per use, from what I can see the * 0.3 multiplier is just to turn it off

Reply:
This was lowered to 750 from 1000 in my latest commit, I forgot to change the comments.